# Translation of Odoo Server.
# This file contains the translation of the following modules:
# * partner_firstname
#
# Translators:
# OCA Transbot <transbot@odoo-community.org>, 2018
msgid ""
msgstr ""
"Project-Id-Version: Odoo Server 11.0\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2018-01-03 20:26+0000\n"
"PO-Revision-Date: 2018-08-22 13:24+0000\n"
"Last-Translator: Rudolf Schnapka <rs@techno-flex.de>\n"
"Language-Team: German (https://www.transifex.com/oca/teams/23907/de/)\n"
"Language: de\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: \n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Generator: Weblate 3.1.1\n"

#. module: partner_firstname
#: code:addons/partner_firstname/models/res_users.py:40
#: code:addons/partner_firstname/models/res_users.py:42
#, python-format
msgid "%s (copy)"
msgstr ""

#. module: partner_firstname
#: model:ir.model,name:partner_firstname.model_res_config_settings
#, fuzzy
msgid "Config Settings"
msgstr "res.config.settings"

#. module: partner_firstname
#: model:ir.model,name:partner_firstname.model_res_partner
msgid "Contact"
msgstr "Kontakt"

#. module: partner_firstname
#: sql_constraint:res.partner:0
msgid "Contacts require a name."
msgstr "Kontakte erfordern einen Namen."

#. module: partner_firstname
#: code:addons/partner_firstname/exceptions.py:10
#, python-format
msgid "Error(s) with partner %d's name."
msgstr "Fehler beim Namen des Partners %d ."

#. module: partner_firstname
#: model:ir.model.fields,field_description:partner_firstname.field_res_partner__firstname
#: model:ir.model.fields,field_description:partner_firstname.field_res_users__firstname
msgid "First name"
msgstr "Vorname"

#. module: partner_firstname
#: model:ir.model.fields,field_description:partner_firstname.field_res_partner__lastname
#: model:ir.model.fields,field_description:partner_firstname.field_res_users__lastname
msgid "Last name"
msgstr "Nachname"

#. module: partner_firstname
#: model:ir.model.fields,field_description:partner_firstname.field_res_partner__name
#: model:ir.model.fields,field_description:partner_firstname.field_res_users__name
msgid "Name"
msgstr ""

#. module: partner_firstname
#: code:addons/partner_firstname/exceptions.py:7
#, python-format
msgid "No name is set."
msgstr "Kein Name eingetragen."

#. module: partner_firstname
#: model:ir.model.fields,help:partner_firstname.field_res_config_settings__partner_names_order
msgid "Order to compose partner fullname"
msgstr "Reihenfolge den vollen Namen des Partners zusammenzustellen"

#. module: partner_firstname
#: model_terms:ir.ui.view,arch_db:partner_firstname.view_general_configuration
#, fuzzy
msgid "Partner Names Order"
msgstr "Reihenfolge Partner-Namensbestandteile"

#. module: partner_firstname
#: model:ir.model.fields,field_description:partner_firstname.field_res_config_settings__partner_names_order_changed
msgid "Partner Names Order Changed"
msgstr "Reihenfolge der Namensbestandteile geändert"

#. module: partner_firstname
#: model:ir.model.fields,field_description:partner_firstname.field_res_config_settings__partner_names_order
msgid "Partner names order"
msgstr "Reihenfolge Partner-Namensbestandteile"

#. module: partner_firstname
#: model_terms:ir.ui.view,arch_db:partner_firstname.view_general_configuration
msgid "Recalculate names"
msgstr "Namen neu ermitteln"

#. module: partner_firstname
#: model_terms:ir.ui.view,arch_db:partner_firstname.view_general_configuration
msgid ""
"Recalculate names for all partners. This process could take so much time if "
"there are more than 10,000 active partners"
msgstr ""
"Namen für alle Partner neu bestimmen. Dieser Prozess kann sehr viel Zeit in "
"Anspruch nehmen, wenn es mehr als 10.000 aktive Partner gibt"

#. module: partner_firstname
#: model:ir.model,name:partner_firstname.model_res_users
msgid "Users"
msgstr "Anwender"
